projects/public-search/src/app/patron-profile/patron-profile.component.html

```diff
@@ -0,0 +1,68 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2021 RERO
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<ngx-spinner type="ball-spin-clockwise" size="medium"></ngx-spinner>
+<ng-container *ngIf="user">
+  <header class="row col-12 ml-0">
+    <i class="fa fa-user fa-3x"></i>
+    <h3 class="ml-3 mt-2">{{ fullname }}</h3>
+  </header>
+  <section class="mt-2">
+    <public-search-patron-profile-message></public-search-patron-profile-message>
+  </section>
+  <section class="mt-4">
+    <tabset id="profile-tabs">
+      <tab>
+        <ng-template tabHeading>
+          {{ 'Loans' | translate }}
+          <span class="badge badge-light">{{ tabs.loan.count }}</span>
+        </ng-template>
+        <public-search-patron-profile-loans [count]="tabs.loan.count"></public-search-patron-profile-loans>
+      </tab>
+      <tab (selectTab)="selectTab('request')">
+        <ng-template tabHeading>
+          {{ 'Requests' | translate }}
+          <span class="badge badge-light">{{ tabs.request.count }}</span>
+        </ng-template>
+        <public-search-patron-profile-requests></public-search-patron-profile-requests>
+      </tab>
+      <tab (selectTab)="selectTab('fee')">
+        <ng-template tabHeading>
+          {{ 'Fees' | translate }}
+          <span *ngIf="feeTotal > 0" class="badge badge-danger">{{ feeTotal | currency:user.organisation.currency:true }}</span>
```

Comment:
check with PO (@iGormilhit @pronguen) :: we use the `badge-warning font-weight-normal` into the professional interface and reserve "danger" to error message (or transaction event due amount)

Reply:
Changed